commons/src/test/java/org/frankframework/util/ClassUtilsTest.java

```diff
@@ -37,21 +38,56 @@ public void testConvertToType() {
 		);
 	}
 
-	private static class DummyClassWithSetter {
+	private class DummyClassWithSetter {
 		private @Getter @Setter String field;
+		private @Getter Object[] testEnums;
```

Comment:
Ik zou verwachten dat dit `TestEnum[]` is en geen Object

Reply:
Oeps, dank. Pas ik aan.